morse.rb

```diff
@@ -0,0 +1,43 @@
+def char_list(char)
+  characters = {
+    '.-' => 'A',
+    '-...' => 'B',
+    '-.-.' => 'C',
+    '-..' => 'D',
+    '.' => 'E',
+    '..-.' => 'F',
+    '--.' => 'G',
+    '....' => 'H',
+    '..' => 'I',
+    '.---' => 'J',
+    '-.-' => 'K',
+    '.-..' => 'L',
+    '--' => 'M',
+    '-.' => 'N',
+    '---' => 'O',
+    '.--.' => 'P',
+    '--.-' => 'Q',
+    '.-.' => 'R',
+    '...' => 'S',
+    '-' => 'T',
+    '..-' => 'U',
+    '...-' => 'V',
+    '.--' => 'W',
+    '-..-' => 'X',
+    '-.--' => 'Y',
+    '--..' => 'Z'
+  }
+  characters[char]
+end
+
+def decode_word(word)
+  words = word.split
+  words.map { |letter| char_list(letter) }.join
+end
+
+def decode_sentense(sentense)
```

Comment:
[OPTIONAL]

- I think you wanted to write `sentence`, but there is a typo. I am mentioning this just to aware otherwise code is working fine.

Reply:
Noted. Thanks